client/components/App.jsx

```diff
@@ -34,6 +36,9 @@ class App extends Component {
   };
 
   componentDidMount() {
+    fetch(`codestitch?email=${codestitchEmail}&password=${codestitchPassword}`)
```

Comment:
Without clearly understanding codestitch.io (can't seem to login), is the argument's beginning syntax correct?